README.md

```diff
@@ -1,2 +1,4 @@
 # testrepo2
 for more testing
+
+this is a test change
```

Comment:
What do you think about this?

Reply:
- [ ] I think you should capitalize "this"

Reply:
Are those really the only ones that need it?

- [x] What about all of the BGC variables?
- [ ] Also, it seems like, in principle, some 2-d variables in accumulMod could need this. For example, what if you are accumulating the 10-day running average of H2OSOI_LIQ: then you would want scale_by_thickness to be true for that accumulated variable, too, right? It seems like handling this correctly would require passing scale_by_thickness into `init_accum_field`: that could be an optional argument that is required if numlev > 1.

Reply:
Are those really the only ones that need it?

- [x] What about all of the BGC variables?
- [ ] Also, it seems like, in principle, some 2-d variables in accumulMod could need this. For example, what if you are accumulating the 10-day running average of H2OSOI_LIQ: then you would want scale_by_thickness to be true for that accumulated variable, too, right? It seems like handling this correctly would require passing scale_by_thickness into `init_accum_field`: that could be an optional argument that is required if numlev > 1.

Reply:
Are those really the only ones that need it?

- [ ] What about all of the BGC variables?
- [ ] Also, it seems like, in principle, some 2-d variables in accumulMod could need this. For example, what if you are accumulating the 10-day running average of H2OSOI_LIQ: then you would want scale_by_thickness to be true for that accumulated variable, too, right? It seems like handling this correctly would require passing scale_by_thickness into `init_accum_field`: that could be an optional argument that is required if numlev > 1.